src/bytes.rs

```diff
@@ -200,6 +202,69 @@ impl Bytes {
         }
     }
 
+    /// Create [Bytes] with a buffer whose lifetime is controlled
+    /// via an explicit owner.
+    ///
+    /// A common use case is to zero-copy construct from mapped memory.
+    ///
+    /// ```ignore
```

Comment:
Instead of marking this test `ignore`, you can add a doc-hidden module called `memmap2` to simulate that the crate exists when compiling the example.

Reply:
I couldn't quite get that to work, as the `Mmap` type would otherwise be found via `use bytes::memmap2::Mmap` and not `memmap2::Mmap`, so I inlined the mock module instead.